File: ads/src/main/java/com/quyunshuo/ads/MainActivity.java
package com.quyunshuo.ads;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.google.android.gms.ads.AdRequest;
import com.google.android.gms.ads.AdView;
import com.google.android.gms.ads.MobileAds;

public class MainActivity extends AppCompatActivity {
    private AdView mAdView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        //初始化广告
        MobileAds.initialize(this, "ca-app-pub-3940256099942544~3347511713");
        mAdView = findViewById(R.id.adView);
        //请求广告
        AdRequest adRequest = new AdRequest.Builder().build();
        //加载广告
        mAdView.loadAd(adRequest);
    }
}

File: Thread/src/main/java/com/quyunshuo/thread/MainActivity.kt
package com.quyunshuo.thread

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        initThread()
    }

    private fun initThread() {
        MainThread().run()
        MainThread2().run()
        MainThread3().call()
    }
}

